tests/unit/test_embedding_models/test_sentence_embedding_model.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+# Date    : 20.07.2021 
+# Author  : Stratulat Ștefan
+# File    : test_sentence_embedding_model.py
+# Software: PyCharm
+from sem_covid.adapters.abstract_model import SentenceEmbeddingModelABC, WordEmbeddingModelABC
+from sem_covid.adapters.embedding_models import (AverageSentenceEmbeddingModel, Word2VecEmbeddingModel,
+                                                 BasicTokenizerModel, SpacyTokenizerModel, TfIdfSentenceEmbeddingModel,
+                                                 UniversalSentenceEmbeddingModel,
+                                                 EurLexBertSentenceEmbeddingModel)
+from tests.unit.conftest import nlp
+from tests.unit.test_embedding_models.fake_keyed_vectors import FakeKeyedVectors
+
+SENTENCES = [
+    'Rather than having all of its functionality built into its core.',
+    'Python was designed to be highly extensible (with modules).',
+    'Rather than having all of its functionality built into its core.',
+    'This compact modularity has made it particularly popular as a means of adding programmable interfaces to existing applications.',
+    "Van Rossum's vision of a small core language with a large standard library and easily extensible interpreter stemmed from his frustrations with ABC, which espoused the opposite approach."
+]
+
+
+def check_sentence_embedding_model(sentence_embedding_model: SentenceEmbeddingModelABC):
```

Comment:
I like helper test functions.
They are best placed in `conftest.py` to maximise reuse. 

Unless they are very very local :)